meta-genivi-dev/recipes-devtools/python/python-pyqt_5.3.1.bb

```diff
@@ -99,12 +106,14 @@ do_install() {
     install -d ${D}${datadir}/sip/qt/
     install -d ${D}${bindir}
     install -d ${D}${libdir}/${PYTHON_DIR}/site-packages/dbus/mainloop
+
```

Comment:
Fixing formatting is always a welcomed change but they shouldn't be hidden in unrelated patches.